src/exampleTypes/TagsText.js

```diff
@@ -34,26 +33,32 @@ let Text = ({
         onChange={e => tree.mutate(node.path, { operator: e.target.value })}
         options={operatorOptions}
       />
-      <TagsInput
-        splitCommas
-        tags={node.values}
-        onTagClick={tag => {
-          F.on(open)()
-          setSelectedTag(tag)
-        }}
-        addTag={tag => {
-          tree.mutate(node.path, { values: [...node.values, tag] })
-        }}
-        removeTag={tag => {
-          tree.mutate(node.path, {
-            values: _.without([tag], node.values),
-          })
-        }}
-        tagStyle={bgJoin(tagToGroupJoin(node.join))}
-        submit={tree.triggerUpdate}
-        placeholder={placeholder}
-      />
-      <Popover open={open}>
+
+      <Popover
+        trigger={
```

Comment:
I think that changes the tags text behavior, 

The Popover should show up when the user clicks the tag instead of the trigger by clicking the input.